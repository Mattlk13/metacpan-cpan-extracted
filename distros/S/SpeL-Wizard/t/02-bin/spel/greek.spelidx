format|ogg
audiodir|spel
chunkdir|spel
macpp|ds|-NoValue-|-NoValue-|
language|english
chunk|greek
