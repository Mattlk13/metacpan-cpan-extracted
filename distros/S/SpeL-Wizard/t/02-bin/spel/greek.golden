 "T" ,equals, the summation for "k" , of: "T" "k" times δ "k", over δ
